src/scrabble.js

```diff
@@ -1,5 +1,183 @@
-function scrabble() {
 
+//Core Scrabble: nailed it
+let newWordArr = []
+let sum = 0
```

Comment:
Nice idea to have this variable outside the other functions. Be carefull about global variables tho, since it can be source of errors in more complex programs